### PR TITLE
adding more staking FAQs

### PR DESCRIPTION
**Pull request by vishalchangrani**
Adding some more staking related FAQs documented here: https://www.notion.so/dapperlabs/c2acf7e6b7a74edd8f72b5db09b58f52?v=3c979316e5cf4b8badd50a84baf7c0bb

## Description

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: docs/content/staking/faq.mdx [hunk 1]
```diff
@@ -27,3 +27,35 @@ However, currently there is a restriction in place from the token lockup rules s
 We have the same restriction in place for an account address and an active delegation.
 This means that 1 account address can have at maximum 1 active stake and 1 active delegation at any given time.
 If a token holder wants to have additional stake and/or delegate transactions, then they must create additional account addresses to do so.
+
+### Is there a utility to generate the generate the FLOW addresses?
```
Comment (joshuahannan) [inline docs/content/staking/faq.mdx:31]:
```suggestion
### Is there a utility to generate the FLOW addresses?
```

File: docs/content/staking/faq.mdx [hunk 2]
```diff
@@ -27,3 +27,35 @@ However, currently there is a restriction in place from the token lockup rules s
...
+A node operator generates the staking and networking keys separately which will be used on the node.
+
+### Can I stake through multiple accounts to meet the minimum FLOW required for staking a node?
+
+No, the minimum staking FLOW should come from a single account for all node type except for the consensus node.
+The minimum for consensus nodes can cumulatively come from a combination a staking account and a delegation account.
+
+### How can I reach the Consensus node minimum stake of 500K FLOW
+
+The consensus node min 500K FLOW can be reached cumulatively with any combination of a lease and delegation. e.g. lease 250K and then delegate the remaining 250K.
```
Comment (joshuahannan) [inline docs/content/staking/faq.mdx:48]:
```suggestion
The consensus node min 500K FLOW can be reached cumulatively with any combination of a lease and delegation as long as the lease is a minimum of 250k FLOW. e.g. lease 250K and then delegate the remaining 250K.
```

File: docs/content/staking/faq.mdx [hunk 3]
```diff
@@ -27,3 +27,35 @@ However, currently there is a restriction in place from the token lockup rules s
...
+
+### Is rewards payout another spork?
+
+No, rewards payout is not a spork but (currently) a separate transaction that we initiate.
+
+### Are leased tokens technically owned by the recipient or are they still considered leased after being staked?
+
+DapperLabs has possession of the private keys for the account where the leased tokens are held, so technically DapperLabs has ownership of them.
+The recipient of the leased tokens however has the ability to stake them for their nodes, but don't have the ability to withdraw them or do anything else with them.
```
Comment (joshuahannan) [inline docs/content/staking/faq.mdx:57]:
```suggestion
The recipient of the leased tokens however has the ability to stake them for their nodes and withdraw the rewards from staking, but doesn't have the ability to withdraw them or do anything else with them.
```

File: docs/content/staking/faq.mdx [hunk 4]
```diff
@@ -27,3 +27,35 @@ However, currently there is a restriction in place from the token lockup rules s
...
+
+### How long will tokens be leased to us?
```
Comment (joshuahannan) [inline docs/content/staking/faq.mdx:59]:
```suggestion
### How long are tokens leased?
```

**Review by joshuahannan — APPROVED** [review]
Looks good! I have a few suggestions.

**Review by laynelafrance — CHANGES_REQUESTED** [review]
committed changes

**Review by 10thfloor — APPROVED** [review]

**Comment by vishalchangrani** [comment]
Closing this PR in favour or https://github.com/onflow/flow/pull/250 but retaining all the commits and changes.

**Pull request closed without merging**